src/Components/DetailPages/DetailPages.scss

```
@@ -0,0 +1,286 @@
+@import "../../Styles/common.scss";
+
+.totalContainer {
+  margin: 0;
+  padding: 0;
+  .firstContainer {
+    width: 1250px;
+    margin: 0 auto;
+    .titleBox {
+      width: 100%;
+      height: 120px;
+      margin-top: 100px;
+      .title {
+        display: flex;
+        position: relative;
+        flex-direction: row;
+        justify-content: space-between;
```

css를 작성하실 때에도 아래와 같은 순서로 작성하시면 더 깔끔한 코드가 됩니다. 아래 css 도 다 수정해주세요!

1. 레이아웃에 큰 영향을 주는 것부터 (레이아웃에 가장 큰 영향을 주는 width, height부터)
2. 비슷한 또는 연관성있는 속성끼리 같이 배열해주세요. 

```suggestion
        display: flex;
        flex-direction: row;
        justify-content: space-between;
        position: relative;
```

---

넵 감사합니다!!